packages/automerge-repo/src/Repo.ts

```diff
@@ -505,6 +505,31 @@ export class Repo extends EventEmitter<RepoEvents> {
       return this.storageSubsystem.id()
     }
   }
+
+  /**
+   * Waits for Repo to finish write changes to disk.
+   * @deprecated
+   * @param documents - if provided, only waits for the specified documents
+   * @param timeout - if provided, the maximum time to wait in milliseconds
+   * @returns Promise<void>
+   */
+  async flush(documents?: DocumentId[], timeout?: number): Promise<void> {
+    if (!this.storageSubsystem) {
+      throw new Error("flush called without storage subsystem")
+    }
+    const handles = documents
+      ? documents.map(id => this.#handleCache[id])
+      : Object.values(this.#handleCache)
+    return Promise.all(
+      handles.map(async handle => {
+        const doc = handle.docSync()
+        if (!doc) {
+          return
+        }
+        return this.storageSubsystem!.flush(handle.documentId, doc, timeout)
+      })
+    ).then(() => {})
```

Comment:
```suggestion
    ).then(() => {})
```
May I ask what this line is for?